### ISSUE TITLE
[BUG] Hybrid mode not switching to spreadCycle when doing mesh bed leveling

### ISSUE THREAD

**Issue by Ivomola** [MarlinFirmware/Marlin#18989]
I'm using Marlin 2.0.6 on a board I recently got, the Fystec Cheetah 1.2a, for the Ender 3. I know, it's not the best, but it is what was nearest on Wallapop (popular Spanish marketplace). The issue I'm facing is that I have Mesh Bed Leveling enabled, as I did in the previous Creality Melzi 1.1.3 that came with my printer, only difference being that that board had Allegro A4988 drivers and Marlin 2.0.5.4, and this one has TMC2208. I don't know what did I change or do differently, but when I start the bed leveling procedure, the speed at which the axis try to move is so high that they either skip, rattle and don't move at all, or move stupidly fast. And not only I can't change the feedrate anywhere, or at least I can't find the option anywhere, but (and this is the bug I want to report), when I disable stealthChop so the drivers use spreadCycle, it still moves very fast, but the drivers handle it, so I enabled HYBRID_THRESHOLD in configuration_adv.h, but no matter how low I set the speed to change to spreadCycle, in the bed leveling wizard, the drivers stay in stealthChop mode. If I set it low enough, when homing they change, when moving via Gcode they change, it's very noticeable because when spreadCycle is on a high pitched noise comes out of the coils of the steppers, but when doing the bed leveling procedure, they stay quiet, and almost all of the time they lose steps and don't move at all. So is there any workaround for it, or for now do I just disable stealthChop when leveling and enable it everywhere else? I'll attatch my configuration files for people to investigate. The language is set to Spanish, tho, so remember to change the language to english in the configuration.h file before using it, unless you know the language.
Tl;dr: HYBRID_THRESHOLD doesn't work when doing a mesh bed leveling.
Thanks!
[Configuration.zip](https://github.com/MarlinFirmware/Marlin/files/5053859/Configuration.zip)

**Comment by sjasonsmith** [comment]
I am going to close this as a configuration issue. Your best option is to go to one of the several support forums available and seek some individuals to help you configure the board properly.

If you do find there is a bug to report, please be sure to follow the issue template so that it is easy for developers to know exactly what the issue is and how to reproduce it.

Some ideas before I close this:
1. Make sure you start with the example configuration for this board. One is already available for an Ender 3:
https://github.com/MarlinFirmware/Configurations/tree/release-2.0.6/config/examples/Creality/Ender-3/FYSETC%20Cheetah%201.2

2. It seems there is likely an issue with TMC communication, so your 256 microsteps are not being configured properly. This would generate extremely fast movement since `DEFAULT_AXIS_STEPS_PER_UNIT` assumes 256 microsteps. Most experienced users will recommend not actually using 256 microsteps anyway. It is a very high load on the CPU.

This Issue Queue is for Marlin bug reports and development-related issues, and we prefer not to handle user-support questions here. (As noted on [this page](https://github.com/MarlinFirmware/Marlin/blob/2.0.x/.github/contributing.md#i-dont-want-to-read-this-whole-thing-i-just-have-a-question).) For best results getting help with configuration and troubleshooting, please use the following resources:

- MarlinFW.org [Marlin Documentation](http://marlinfw.org/)
- RepRap.org [Marlin Forum](http://forums.reprap.org/list.php?415)
- [Tom's 3D Forums](https://forum.toms3d.org/)
- Facebook Group ["Marlin Firmware"](https://www.facebook.com/groups/1049718498464482/)
- Facebook Group ["Marlin Firmware for 3D Printers"](https://www.facebook.com/groups/3Dtechtalk/)
- [Marlin Configuration](https://www.youtube.com/results?search_query=marlin+configuration) on YouTube
- Marlin Discord server. Join link: https://discord.gg/n5NJ59y

After seeking help from the community, if the consensus points to a bug in Marlin, then you should post a [bug report](https://github.com/MarlinFirmware/Marlin/issues/new/choose).
